tst/teststandard/grpprmcs.tst

```diff
@@ -312,6 +312,10 @@ Group
 Group
 gap> List( ChiefSeriesOfGroup( g ), Size );
 [ 1215506, 607753, 31987, 1103, 1 ]
+gap> u:=NormalClosure(g,SylowSubgroup(g,2));;
+gap> cs:=ChiefSeriesThrough(g,[u]);;
+gap> List(cs,Size);
+[ 1215506, 63974, 2206, 1103, 1 ]
```

Comment:
Thanks @hulpke. Sorry, I didn't notice that your new four-argument `ContainedConjugates` was already tested by the manual example you added. 🙂 

As for this test, since it's `CompositionSeriesThrough` rather than `ChiefSeriesThrough` that is added by this pull request, would you be happy to accept this code suggestion? Unsurprisingly the series are identical, but from my quick look at things, only the latter uses your new `CompositionSeriesThrough` code. Please correct me if I'm mistaken.
```suggestion
[ 1215506, 63974, 2206, 1103, 1 ]
gap> cs:=CompositionSeriesThrough(g,[u]);;
gap> List(cs,Size);
[ 1215506, 63974, 2206, 1103, 1 ]
```

Reply:
Thank you, @wilfwilson . Indeed it should have been `CompositionSeriesThrough` and I mistyped and did not spot it (since it returned a composition series). Will add this.